migrations/20200117094246-add-url-at-to-build.js

```diff
@@ -0,0 +1,5 @@
+exports.up = (db, callback) =>
+  db.addColumn('build', 'url', { type: 'string' }, callback);
```

Comment:
Will we need to insert the `url` based on the site's `awsBucketName` column and the AWS bucket url.

ie `<awsBucketName><S3 Bucket Region>.amazonaws.com`.

Reply:
if i understand your statement correctly ...  yes, because of the way we are using the serializers .... it seems so ... let me know if you still see otherwise .... plus @davemcorwin feels very strongly about storing the deployment urls in the database

Reply:
@apburnes Are you asking whether we need to save THIS url (the one with the bucket name) vs. what's called the `viewLink` or the prettier one?

Reply:
My bad on `<S3 Bucket Region>.amazonaws.com`, dismiss that...

The migration is not populating the old builds with a `url` for either the corresponding `<awsBucketName>.app.cloud.gov` or `federalist-proxy.app.cloud.gov`. Is this happening in the serializer and I'm just missing it?

Reply:
we agreed we would update the old urls before the bucket migration to backfill those values